composer.json

```diff
@@ -34,15 +34,17 @@
 		"react/socket": "^1.12",
 		"manticoresoftware/telemetry": "^0.1.19",
 		"symfony/dependency-injection": "^6.1",
-		"manticoresoftware/buddy-core": "dev-main",
```

Comment:
The workers branch was merged into the main branch, but before we continue, we should revert this merge and move the changes back to the dev-main branch.